service/templates/admin.html

```diff
@@ -1408,6 +1408,10 @@ <h2 class="modal-title">[[modalTitle]]</h2>
 										<label>API Key</label>
 										<input type="text" ng-model="Settings.cedar.api_key">
 									</md-input-container>
+									<md-input-container class="control" style="width:45%;">
+										<label>DB URL</label>
+										<input type="text" ng-model="Settings.cedar.db_url">
+									</md-input-container>
```

Comment:
Is there supposed to be a setting for the DB name? Would it overwrite/delete the DB name if some other setting in this section was saved?

Reply:
Ah yes, I had been hard coding settings in the staging DB so didn't add a setting for the name yet. Fixed.